### PR TITLE
ZWEB-3674 JF Make All Library Packages Fixed & Fix Security Issues

### PR DESCRIPTION
**Pull request by Jacob-Friesen**
**This library has no test files for some reason. Despite this I still updated all the test infrastructure which currently does nothing. So a very careful manual review is necessary taking into account all the changes made.**

**Replication**
1. Run the install and audit: `npm install && npm audit`
1. Run the tests: `npm test`
1. Confirm that all the gulp tasks still run

Note a lot of unneeded functionality like karma PhantomJS or automatic JS file wrapping is no longer needed, so it was removed. Also, there is still one high dependency due to a Karma security issue reported today (https://www.npmjs.com/advisories/1486). This can either be removed temporarily since no tests are needed or ignored until the fix it (likely soon).

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 z-open
+Copyright (c) 2020 Zimit
```
Comment (etelmon) [inline LICENSE:3]: This branch is based out of dev, but I see that there were 2 releases that were not merged to dev. Pls checkout

Comment (Jacob-Friesen) [reply LICENSE:3]: I don't know anything about those. The developers should have merge those changes into `develop` if they are being used in our code. Anyways, I will contact the original developers and try to deal with those issues.

Comment (Jacob-Friesen) [reply LICENSE:3]: Ok. I have opened a PR of that branch and will properly review it and post the review link here. PR: https://github.com/z-open/zerv-ng-security/pull/2

Comment (Jacob-Friesen) [reply LICENSE:3]: Here the the PR review with comments for your to address: https://github.com/z-open/zerv-ng-security/pull/2#pullrequestreview-412776198

File: package.json [hunk 2]
```diff
@@ -6,25 +6,24 @@
   "repository": {
     "url": ""
   },
-  "version": "0.0.3",
+  "version": "0.0.4",
   "dependencies": {
-    "angular": "~1.5.8",
+    "angular": "1.7.9",
     "lodash": "4.17.15",
     "zerv-ng-sync": "z-open/zerv-ng-sync#1.2.9"
```
Comment (etelmon) [inline package.json:13]: 1.2.11?

File: LICENSE (hunk 1 above)
Comment (etelmon) [reply LICENSE:3]: Done

File: package.json (hunk 2 above)
Comment (Jacob-Friesen) [reply package.json:13]: This one now has an incremented version.

**Pull request merged**